pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -669,6 +669,10 @@ type MirrorHealthCheckSpec struct {
 	// +optional
 	// +nullable
 	Mirror HealthCheckSpec `json:"mirror,omitempty"`
+	// +optional
```

Comment:
Did you mean to add these probes to the `MetadataServerSpec` type instead?

Reply:
Yes right `MetadataServerSpec`  would be right place